src/main/kotlin/com/team4099/robot2022/auto/AutonomousSelector.kt

```diff
@@ -0,0 +1,43 @@
+package com.team4099.robot2022.auto
+
+import com.team4099.lib.pathfollow.trajectoryFromPathfinder
+import com.team4099.robot2022.commands.drivetrain.AutoDriveCommand
+import com.team4099.robot2022.subsystems.Drivetrain
+import edu.wpi.first.math.geometry.Rotation2d
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser
+import edu.wpi.first.wpilibj2.command.Command
+
+class AutonomousSelector {
```

Comment:
instead of doing a companion object, can we just make this a singleton class (or whatever its called in Kotlin when you do `object` instead of `class` like we do for all of our subsystems